app/views/affiliates/index.html.haml

```diff
@@ -1,21 +1,23 @@
 = content_for :h1 do
-  = t("pages.global_settings")
+  = text("pages.global_settings")
 
 = content_for :sidebar do
   = render "admin/shared/sidenav_global", sidenav_tab: "affiliates"
 
-%h2= t("pages.affiliates")
-= link_to t("affiliates.add"), new_affiliate_path, class: "btn-add"
+%h2= text("pages.affiliates")
+= link_to text("affiliates.add"), new_affiliate_path, class: "btn-add"
 
 %table.table.table-striped.table-hover
   %thead
     %tr
       %th
-      %th{width: "100%"}= t("affiliates.th")
+      %th= Affiliate.model_name.human
+      %th= Affiliate.human_attribute_name(:subaffiliates_enabled)
```

Comment:
I don't know how useful displaying this flag on the index is. It would not take much to remove.